src/Services/Passenger/src/Passenger/Passengers/Models/ValueObjects/PassportNumber.cs

```diff
@@ -0,0 +1,67 @@
+namespace Passenger.Passengers.Models.ValueObjects;
+using System;
+
+
+//public record PassportNumber : GenericValueObject<string>
```

Comment:
Please remove commented code in your code.

Reply:
done